resources/devices/mobile-phone/htc.json

```diff
@@ -160,7 +160,7 @@
         "properties": {
             "Device_Type": "Mobile Phone",
             "Device_Code_Name": "801E",
-            "Device_Name": "One M7",
+            "Device_Name": "One M7 LTE",
```

Comment:
There's an 801N apparently that isn't LTE (just 3G), so I figured it would be useful to add this in the name.